#! /bin/sh -e

# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: localedata header fix
# DP: Author: Jordi Mallach <jordi@debian.org>
# DP: Upstream status: In CVS
# DP: Status Details:
# DP: Date: 2003-08-27

if [ $# -ne 2 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -d "$2" -f --no-backup-if-mismatch -p0 < $0;;
    -unpatch) patch -d "$2" -f --no-backup-if-mismatch -R -p0 < $0;;
    *)
	echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
	exit 1
esac
exit 0

# append the patch here and adjust the -p? flag in the patch calls.
Index: locales/an_ES
===================================================================
RCS file: /cvs/glibc/libc/localedata/locales/an_ES,v
retrieving revision 1.2
retrieving revision 1.3
diff -u -r1.2 -r1.3
--- localedata/locales/an_ES	26 Jun 2003 07:29:02 -0000	1.2
+++ localedata/locales/an_ES	25 Aug 2003 08:27:32 -0000	1.3
@@ -26,10 +26,10 @@
 email      "bug-glibc@gnu.org"
 tel        ""
 fax        ""
-language   "an"
-territory  "ES"
-revision   "1.0"
-date       "2003-06-15"
+language   "Aragonese"
+territory  "Spain"
+revision   "1.1"
+date       "2003-08-25"
 %
 category  "i18n:1999";LC_IDENTIFICATION
 category  "i18n:1999";LC_CTYPE
Index: locales/wa_BE
===================================================================
RCS file: /cvs/glibc/libc/localedata/locales/wa_BE,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- localedata/locales/wa_BE	23 Feb 2002 19:29:34 -0000	1.1
+++ localedata/locales/wa_BE	25 Aug 2003 18:35:41 -0000	1.2
@@ -11,8 +11,8 @@
 address ""
 tel ""
 fax ""
-language "wa"
-territory "BE"
+language "Walloon"
+territory "Belgium"
 revision "0.7"
 date "2002-02-11"
 
Index: locales/yi_US
===================================================================
RCS file: /cvs/glibc/libc/localedata/locales/yi_US,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- localedata/locales/yi_US	3 Aug 2001 08:39:48 -0000	1.1
+++ localedata/locales/yi_US	25 Aug 2003 18:35:41 -0000	1.2
@@ -11,8 +11,8 @@
 email "pablo@mandrakesoft.com"
 tel ""
 fax ""
-language "yi"
-territory "US"
+language "Yiddish"
+territory "USA"
 revision "0.3"
 date "2001-01-28"
 
